docs: Studio setup to include `&studio_unlock`

petejohanson opened the pull request:
* Add a section about adding `&studio_unlock` to the stock keymap.

caksoylar commented on docs/docs/development/hardware-integration/studio-setup.md:
```diff
@@ -138,3 +140,8 @@ Here is a fixed up Corne mapping:
+
+# Studio Lock Keymap Update
+
+The stock keymap should be updated to include some ability to unlock the device for studio updates, by adding `&studio_unlock` to the
```
```suggestion
The stock keymap should be updated to include a way to unlock the device for studio updates, by adding a [`&studio_unlock`](../../keymaps/behaviors/studio-unlock.md) binding to the
```

caksoylar commented on docs/docs/development/hardware-integration/studio-setup.md:
```diff
@@ -138,3 +140,8 @@ Here is a fixed up Corne mapping:
 
 Notice how the outer column positions in the 6-column layout are mapped to positions 36, 37, etc. in the 5-column layout. The 5-column layout only uses key positions up to 35, so those bindings in the outer columns will get migrated into the "extra space" that is ignored by the smaller layout, preserved to get mapped back in place when the user switches back.
+
+# Studio Lock Keymap Update
```
```suggestion
# Studio Unlock Keymap Update
```

caksoylar reviewed (commented):
While this is good, can we add a remark to https://zmk.dev/docs/features/studio as well? It's necessary for end users to know as well sincce they'll probably be using non-default keymaps.

petejohanson commented:
> While this is good, can we add a remark to https://zmk.dev/docs/features/studio as well? It's necessary for end users to know as well sincce they'll probably be using non-default keymaps.

Added, and incorporated your suggestions.

caksoylar commented on docs/docs/features/studio.md:
```diff
@@ -10,6 +10,10 @@ ZMK Studio support is in alpha. Although best efforts are being made, keeping co
 
+## Keymap Changes
+
+To unlock you keyboard to allow ZMK Studio to make changes, you'll need to add a [`&studio_unlock`](../keymaps/behaviors/studio-unlock.md) binding to the keymap in some unused location.
```
```suggestion
To unlock your keyboard to allow ZMK Studio to make changes, you'll need to add a [`&studio_unlock`](../keymaps/behaviors/studio-unlock.md) binding to the keymap.
```
"Unused location" seems unneded here since user can decide whether to move things around, etc.

caksoylar reviewed (approved):
Small tweak, otherwise LGTM.
